draft-ietf-httpbis-message-signatures.md

```diff
@@ -480,6 +480,7 @@ The signature parameters component value is the serialization of the signature p
 * `nonce`: A random unique value generated for this signature as a String value.
 * `alg`: The HTTP message signature algorithm from the HTTP Message Signature Algorithm Registry, as a String value.
 * `keyid`: The identifier for the key material as a String value.
+* `context`: A context for the signature as a String value. This value is used by applications to help identify the signature's validity in specific applications or protocols. Context is often needed for avoiding cross-protocol attacks.
```

Comment:
The third sentence was added by @yaronf in [1c081b0](https://github.com/httpwg/http-extensions/pull/2222/commits/1c081b0df5983d58a80a3568c6ae2096fe48132d). I'm actually fine with backing it off from here because the discussion really should be in a larger space about using the context, in security considerations or something like that.